docs/source/substrafl_doc/substrafl_overview.rst

```diff
@@ -90,22 +91,14 @@ Node
 ^^^^
 There are three types of node:
 
+.. warning:: The term 'node' is used interchangeably with 'organization' in Substra. Both terms represent different points in a federated data network.
```

Comment:
I would remove this warning.
In the documentation and in the software there is not confusion between node and organization. However there is often confusion when we speak.